### ISSUE TITLE
pass extra options to docker-compose up

### ISSUE THREAD

**Issue by rhyek** [testcontainers/testcontainers-node#105]
I'm running into the issue described here docker/compose#1487. I need to rebuild an image for a local dockerfile and would like the option pass the `--build` command line argument to docker-compose somehow.

Maybe something like passing a subset of the official [up](https://docs.docker.com/compose/reference/up/) options to the `up` method.

For example:
```ts
const environment = await new DockerComposeEnvironment(composeFilePath, composeFile)
  .up({ build: true });
```

I'm currently having to manually remove the cached docker image for my local project. Might there be another workaround I can use, currently?

**Comment by cristianrgreco** [comment]
Hi @rhyek, do you see any issues with https://github.com/testcontainers/testcontainers-node/pull/106 ?

**Comment by rhyek** [comment]
@cristianrgreco that seems like it would do the trick.

**Comment by cristianrgreco** [comment]
@rhyek Thanks, this is available in 3.1.0, please let me know if it works for you and if it does please close this issue :slightly_smiling_face:

**Comment by rhyek** [comment]
@cristianrgreco it works perfectly. Thank you!
